homeassistant/components/ambient_network/coordinator.py

```diff
@@ -0,0 +1,37 @@
+"""DataUpdateCoordinator for the Ambient Weather Network integration."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import Any
+
+from aioambient import OpenAPI
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN, ENTITY_MAC_ADDRESS, ENTITY_STATIONS, LOGGER
+
+
+class AmbientNetworkDataUpdateCoordinator(DataUpdateCoordinator):
+    """The Ambient Network Data Update Coordinator."""
+
+    def __init__(
+        self, hass: HomeAssistant, api: OpenAPI, scan_interval: timedelta
```

Comment:
scan_interval is static, no need to make it a parameter

Reply:
Fixed.